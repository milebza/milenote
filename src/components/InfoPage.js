import React from 'react'

const InfoPage = () => (
  <div className="form">
    <p className="text-center">Coded and designed by Milena Baeza</p>  
  </div>
)

export default InfoPage
