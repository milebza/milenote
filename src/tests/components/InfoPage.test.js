import React from 'react'
import { shallow } from 'enzyme'
import InfoPage from '../../components/InfoPage'

test('should render InfoPage correctly', () => {
  const wrapper = shallow(<InfoPage />)
  expect(wrapper).toMatchSnapshot()
})
