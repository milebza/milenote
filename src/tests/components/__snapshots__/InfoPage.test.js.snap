// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render InfoPage correctly 1`] = `
<div
  className="info-page"
>
  <div
    className="flex"
  >
    <div
      className="col col-xs-2 text-center"
    >
      <span
        className="info-page__icon icon icon-list2"
      />
    </div>
    <div
      className="col col-xs-10"
    >
      <p
        className="info-page__description"
      >
        See your list of notes. Click on a note to see its full content
      </p>
    </div>
  </div>
  <div
    className="flex"
  >
    <div
      className="col col-xs-2 text-center"
    >
      <span
        className="info-page__icon icon icon-search"
      />
    </div>
    <div
      className="col col-xs-10"
    >
      <p
        className="info-page__description"
      >
        Search by whatever word you want
      </p>
      <p
        className="info-page__description"
      >
        Use the calendar to filter by dates
      </p>
    </div>
  </div>
  <div
    className="flex"
  >
    <div
      className="col col-xs-2 text-center"
    >
      <span
        className="info-page__icon icon icon-file-text2"
      />
    </div>
    <div
      className="col col-xs-10"
    >
      <p
        className="info-page__description"
      >
        Add a new note
      </p>
    </div>
  </div>
  <div
    className="flex"
  >
    <div
      className="col col-xs-2 text-center"
    >
      <span
        className="info-page__icon icon icon-bin"
      />
    </div>
    <div
      className="col col-xs-10"
    >
      <p
        className="info-page__description"
      >
        Remove a note
      </p>
    </div>
  </div>
  <div
    className="flex info-page__bug"
  >
    <div
      className="col col-xs-2 text-center"
    >
      <span
        className="info-page__icon icon icon-pacman"
      />
    </div>
    <div
      className="col col-xs-10"
    >
      <p
        className="info-page__description"
      >
        Found a bug? 
        <a
          className="info-page__link"
          href="https://github.com/milebza/milenote/issues"
          target="_blank"
        >
          Tell me
        </a>
      </p>
    </div>
  </div>
  <p
    className="info-page__credit text-center"
  >
    <small>
      © 2017 — 
      <a
        className="info-page__credit-link"
        href="https://milebza.github.io/"
        target="_blank"
      >
        Milena Baeza
      </a>
    </small>
  </p>
</div>
`;
